python/paddle/fluid/tests/unittests/parallel_executor_test_base.py

```diff
@@ -76,6 +76,7 @@ def run_executor(exe, feed, fetch_list, program=None):
                 exec_strategy.use_experimental_executor = True
 
             build_strategy = fluid.BuildStrategy()
+            build_strategy.debug_graphviz_path = "/tmp/graph_viz"
```

Comment:
use python `tempfile`?

Reply:
removed. This was only used for debugging